addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/WebserviceHandler.java

```diff
@@ -139,6 +139,13 @@ public void updateStationData() {
                 // if the result is not OK, no updates are done and the status of the Bridge goes to OFFLINE!
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
                         "Empty return or no internet connection");
+                // if the Bridge goes OFFLINE, all Stations will go OFFLINE as well
+                for (Thing thing : getThing().getThings()) {
+                    StationHandler tkh = (StationHandler) thing.getHandler();
+                    tkh.updateStatus(ThingStatus.OFFLINE);
+                }
+                // and the scheduled job gets stopped!
+                pollingJob.cancel(true);
```

Comment:
So it cannot be because there is temporary failure at the servers of tankerkonig... That they return a 500 or any other errorcode. Or because they had some scheduled downtime and returned a non-json response stating their maintenance.